PurrSoft/PurrSoft.Api/Controllers/VolunteerController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using AlbumStore.Application.Commands.VolunteerCommands;
+using AlbumStore.Application.Queries.VolunteerQueries;
+using AlbumStore.Application.QueryOverviews;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using PurrSoft.Api.Controllers.Base;
+using PurrSoft.Application.Common;
+using System.Net;
+
+namespace PurrSoft.Api.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class VolunteerController : BaseController
+{
+    [HttpGet("")]
+    [Authorize(AuthenticationSchemes = "Bearer")]
+    [ProducesResponseType(typeof(List<VolunteerOverview>), (int)HttpStatusCode.OK)]
+    public async Task<CollectionResponse<VolunteerOverview>> GetVolunteers([FromQuery] GetFilteredVolunteersQueries query)
```

Comment:
Response type does not correspond with the one that is sent.
You are sending a CollectionResponse that contains a list of volunteer overview and totalRecords yet you have in the response the type List<VolunteerOverview>